### ISSUE TITLE
[BUG] - No ssh keys added to authorized_hosts and no root password in ansible

### ISSUE THREAD

**Issue by goetzk** [vultr/ansible-collection-vultr#35]
**Describe the bug**
I've tried to create an instance using SSH keys but when I log in .ssh/authorized_keys is empty but no password information is returned in ansible.

From /var/log/cloud-init.log:
```
2022-11-13 23:36:41,103 - util.py[DEBUG]: Changing the ownership of /root/.ssh to 0:0
2022-11-13 23:36:41,103 - util.py[DEBUG]: Writing to /root/.ssh/authorized_keys - wb: [600] 0 bytes
2022-11-13 23:36:41,103 - util.py[DEBUG]: Changing the ownership of /root/.ssh/authorized_keys to 0:0
2022-11-13 23:36:41,104 - util.py[DEBUG]: Reading from /root/.ssh/authorized_keys (quiet=False)
2022-11-13 23:36:41,104 - util.py[DEBUG]: Read 0 bytes from /root/.ssh/authorized_keys
2022-11-13 23:36:41,104 - util.py[DEBUG]: Writing to /root/.ssh/authorized_keys - wb: [600] 0 bytes
2022-11-13 23:36:41,105 - util.py[DEBUG]: Reading from /etc/ssh/sshd_config (quiet=False)
2022-11-13 23:36:41,105 - util.py[DEBUG]: Read 3298 bytes from /etc/ssh/sshd_config
2022-11-13 23:36:41,105 - util.py[DEBUG]: Reading from /root/.ssh/authorized_keys (quiet=False)
2022-11-13 23:36:41,105 - util.py[DEBUG]: Read 0 bytes from /root/.ssh/authorized_keys
2022-11-13 23:36:41,105 - util.py[DEBUG]: Writing to /root/.ssh/authorized_keys - wb: [600] 0 bytes
```
and 
```
:~# cat .ssh/authorized_keys 
:~# 
```
In cloud-config I can see password auth being enabled and roots password being set:
```
 "#cloud-config\n{\"package_upgrade\":true,\"disable_root\":false,\"manage_etc_hosts\":true,\"system_info\":{\"default_user\":{\"name\":\"root\"}},\"ssh_pwauth\":1,\"chpasswd\":{\"list\":[\"root:$6$oH6edejpgFjgnkPT$[etc etc]1\"],\"expire\":false}}",
```

**To Reproduce**
1. Build server using Ansible; my instance creation looks like this
```
    - name: Create Instance
      vultr.cloud.instance:
        api_key: "{{ vultr_api_key }}"
        activation_email: "{{ hostvars[instance_hostname].instance_activation_email |default('no') }}"
        enable_ipv6: false
        backups: "{{ hostvars[instance_hostname].instance_backups |default('no') }}"
        firewall_group: "{{ firewall_group_result.vultr_firewall_group.description |default(fallback_vultr_firewall_group) }}"
        hostname: "{{ instance_hostname }}"
        label: "{{ hostvars[instance_hostname].instance_label }}"
        os: "{{ hostvars[instance_hostname].instance_os }}"
        plan: "{{ hostvars[instance_hostname].instance_plan }}"
        region: "{{ hostvars[instance_hostname].deployment_region }}"
        ssh_keys: "{{ hostvars[instance_hostname].instance_ssh_keys }}"
        # startup_script: "{{ hostvars[instance_hostname].startup_script_result }}"
        state: present
      register: instance_result
```
The ssh keys variable looks like this (not that its relevant to the point of this issue):
```
group_vars/all:instance_ssh_keys: [ 'Karl (butter)', 'Monica (laptop)' ]
```
Edit to note: I tried this with and without (); keys were not installed. Should I raise a separate question about that?

2. Try to log in to server using ssh keys, find its not possible
3. Log in to web, get password and find keys have not been installed.

**Expected behavior**
If SSH keys are not correctly installed roots password should be returned via Ansible so the next steps can be done via password if required.

**Additional context**

```
$ ansible-playbook --version
ansible-playbook [core 2.13.4]
  config file = None
  configured module search path = ['/home/kgoetz/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/kgoetz/.venvs/ansible_core/lib/python3.9/site-packages/ansible
  ansible collection location = /home/kgoetz/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/kgoetz/.venvs/ansible_core/bin/ansible-playbook
  python version = 3.9.2 (default, Feb 28 2021, 17:03:44) [GCC 10.2.1 20210110]
  jinja version = 3.1.2
  libyaml = True
```

collection version
```
$ ansible-galaxy collection list |grep vultr
vultr.cloud          1.3.0  

```

**Comment by resmo** [comment]
fixed in v1.3.1

**Comment by goetzk** [comment]
Thanks!
